packages/unified/src/utilsv5.ts

```diff
@@ -265,7 +266,7 @@ export class MDUtilsV5 {
             });
           }
           if (!data.config) {
-            data.config = data.engine!.config;
+            data.config = DConfig.readConfigSync(data.engine!.wsRoot);
```

Comment:
this gets used all over the place. can it use the cache?

Reply:
sure